cms/djangoapps/contentstore/views/preview.py

```diff
@@ -311,6 +311,8 @@ def _studio_wrap_xblock(xblock, view, frag, context, display_name_only=False):
             'content': frag.content,
             'is_root': is_root,
             'is_reorderable': is_reorderable,
+            'is_selected': context.get('is_selected', False),
+            'slectable': context.get('slectable', False),
```

Comment:
Should it be `selectable`? Looks like a typo.

Reply:
updated